Xamarin.Forms.Core/ContentPage.cs

```diff
@@ -22,5 +22,19 @@ protected override void OnBindingContextChanged()
 				SetInheritedBindingContext(content, BindingContext);
 			}
 		}
+
+		internal override void OnControlTemplateChanged(ControlTemplate oldValue, ControlTemplate newValue)
+		{
+			if (oldValue == null)
+				return;
+
+			base.OnControlTemplateChanged(oldValue, newValue);
+			View content = Content;
+			ControlTemplate controlTemplate = ControlTemplate;
+			if (content != null && controlTemplate != null)
```

Comment:
Why the trap for `content == null`? Seems `TemplateUtilities.cs` L122 ensures that's the case. Should we throw instead?